.storybook/main.ts

```diff
@@ -27,7 +27,13 @@ const config = {
       },
     });
   },
-  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  stories: [
+    // Use implicit paths for all stories
+    '../src/**/*.stories.@(js|jsx|ts|tsx)',
+    // For a better structure, need to reorganize some things. Should first confirm approach that "src" has comment element components and "modules" have components for specific pages or modules, which is the vague pattern right now.
+    // - Move src/components/clientDashboard components into module(s)
```

Comment:
Sorry for the disorganization! In a way this definition is a bit unintuitive to me, since "module" maybe implies some standard part that can be reused in different contexts, as in "modular"? But upon researching a little it seems like it's the convention, right? So I am fine with it 👍 

Under that convention I agree with moving `pages/` into `modules/`.

Other misc questions about our app structure/conventions, now that I'm looking at it -- these aren't related to storybook though: 
- What goes in `scripts` vs. in util directories like `src/utils` or `src/modules/form/util`, or even other inlined util files like hmisUtil.ts?
- Should we move  `api/operations` -> `api/` ? (since it's not just operations in there)

Reply:
+1 for conventions. Did you find documentation or other reference for a convention?

Reply:
Yeah we can discuss! This was more of a placeholder note.

The `modules` structure was based off another GR application. Imo it should be similar to `features/` in [bulletproof react](https://github.com/alan2207/bulletproof-react/blob/master/docs/project-structure.md) structure., but we don't follow that super consistently. I can come up with some proposed structure documentation  that we can review together